integrations/api_oauth2_apps_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.package models
+
+package integrations
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"code.gitea.io/gitea/models"
+	api "code.gitea.io/gitea/modules/structs"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAPICreateOAuth2Application(t *testing.T) {
```

Comment:
```suggestion
func TestAPIOAuth2Application(t *testing.T) {
```

Reply:
moved to a private function and a sequence call from `TestOAuth2Application`